templates/options.erb

```diff
@@ -0,0 +1,7 @@
+Dpkg::Options {
+<%- @_options.each_pair do |config, value| 
+  if !!value == value and value then -%>
```

Comment:
well because ruby doesn't have a type bool (true is TrueClass and false is FalseClass) this is the best way i could find[1], happy to recive suggestions

[1]http://stackoverflow.com/questions/3028243/check-if-ruby-object-is-a-boolean